src/groupeddataframe/splitapplycombine.jl

```diff
@@ -453,18 +453,22 @@ julia> combine(gd, :, AsTable(Not(:a)) => sum, renamecols=false)
 ```
 """
 function combine(f::Base.Callable, gd::GroupedDataFrame;
-                 keepkeys::Bool=true, ungroup::Bool=true, renamecols::Bool=true)
+                 keepkeys::Bool=true, ungroup::Bool=true, renamecols::Bool=true,
+                 nthreads::Int=NTHREADS)
```

Comment:
Ideally we should use a keyword argument name that is standard across packages, but for now I haven't found examples of similar cases elsewhere.

We should also anticipate what it will look like once we switch the default to  the automatic optimal number of threads. Would it be `nthreads=nothing`, `nthreads=:auto`, `nthreads=0`? We could also use another name: `threads=true` looks good at it also works with e.g. `threads=5`. Though using a different type forces recompilation, so maybe `0` is better as a default, even though it's not super logical.